src/anyvar/translate/vrs_python.py

```diff
@@ -0,0 +1,65 @@
+"""Normalize incoming variation descriptions with the VRS-Python library."""
+from os import environ
+from typing import Optional
+
+from biocommons.seqrepo import SeqRepo
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy, _SeqRepoDataProxyBase
+from ga4gh.vrs.extras.translator import Translator
+
+from anyvar.utils.types import VrsPythonVariation
+
+from . import _Translator
+
+SEQREPO_ROOT_DIR = environ.get("SEQREPO_ROOT_DIR", "/usr/local/share/seqrepo/latest")
+
+
+class VrsPythonTranslator(_Translator):
+    """Translator layer using VRS-Python Translator class."""
+
+    def __init__(self, seqrepo_proxy: Optional[_SeqRepoDataProxyBase] = None) -> None:
+        """Initialize VRS-Python translator.
+
+        :param seqrepo_proxy: existing SR proxy instance if available. If not given,
+            try to construct a data proxy using local data at the file address
+            defined under the environment variable ``SEQREPO_ROOT_DIR``, or
+            ``/usr/local/share/seqrepo/latest`` by default.
+        """
+        if not seqrepo_proxy:
+            seqrepo_proxy = SeqRepoDataProxy(SeqRepo(SEQREPO_ROOT_DIR))
+        self.tlr = Translator(
+            data_proxy=seqrepo_proxy
+        )
+
+    def translate(self, var: str) -> Optional[VrsPythonVariation]:
+        """Translate provided variation text into a normalized VRS object.
+
+        :param var: user-provided string describing or referencing a variation.
+        :returns: VRS variation object if able to normalize
```

Comment:
Not sure if normalize is the right word for this since it could also fail other checks like regex

```suggestion
        :returns: VRS variation object if able to translate to VRS object
```